clients/admin-ui/cypress/e2e/privacy-requests.cy.ts

```diff
@@ -0,0 +1,37 @@
+describe("Privacy Requests", () => {
```

Comment:
Looks like I introduced conflicts by creating this file: [PR#2473](https://github.com/ethyca/fides/pull/2473/files#diff-ac4af214da8b1b9c4d5dee6d40c5d742100c917ff3928f6520c89bceb1e08653R7). Should be able to just add the the describe blocks b/c they're testing different routes. There's also a place for stubs and fixtures now.

Reply:
Oh nice! I'm waiting for Adam to send me some stubbed responses so I haven't actually written any tests here yet. Appreciate you pointing it out though!